README.md

```diff
@@ -1,8 +1,9 @@
-# Sample GroupMe NodeJS Callback Bot
+# Sample GroupMe NodeJS Bot
+[**Python**](https://github.com/groupme/bot-tutorial-nodejs) | **JavaScript**
```

Comment:
See #33! I have created a Python version of this package which I would love to transfer to the GroupMe GitHub org for public benefit.